rhizome/host/bin/prep_host.rb

```diff
@@ -106,3 +113,27 @@
 
 r "systemctl enable cron"
 r "systemctl start cron"
+
+# setup blocking unused ip addresses
+FileUtils.touch(Util::NFTABLES_PATH)
+block_ip_rules = []
+if (to_block_ip_ranges = JSON.parse(additional_ip_addresses))
```

Comment:
to_block_ip_ranges. I don't understand anything from this name. Is it  ip ranges we want to block, or is it like block of ip ranges, or something else? Can we use more a better name here?

Reply:
Is this better?

Reply:
yep.